modules/system/defaults/dock.nix

```diff
@@ -133,7 +133,13 @@ in {
       apply = value:
         if !(isList value)
         then value
-        else map (app: { tile-data = { file-data = { _CFURLString = app; _CFURLStringType = 0; }; }; }) value;
+        else map (app:
+          if app == "large-spacer"
+          then { tile-data = {}; tile-type = "spacer-tile"; }
+          else if app == "small-spacer"
+          then { tile-data = {}; tile-type = "small-spacer-tile"; }
```

Comment:
Didn't know if another option would be to check against a list of `tile-types` and just forward the app as a tile-type into a single if block